### ISSUE TITLE
substituir o logo atual do mapOS e suas dimensões

### ISSUE THREAD

**Issue by thadeu220** [RamonSilva20/mapos#179]
eu gostaria de saber, como que eu faço para remover o logo padrão do mapOS, e inserir no lugar, a minha logo pessoal da minha loja, assim, ficando mais profissional.
já que, quando o cliente não está com a O.S em mãos., pedimos para que ele insira no nosso sistema mapOS o seu documento.
após ele inserir o documento, é exibido os detalhes.
porém, aquela logo do mapOS no lugar de nossa própria logo, é algo estranho.
queria saber como colocar a nossa própria logo no lugar da logo padrão, achei um arquivo logo dentro de:
/public_html/assets\img\logo.png
é este o arquivo da logo ?
ao trocar essa imagem, a logo do painel é alterada para a nova imagem ?
agradeço!

**Comment by Pr3d4dor** [comment]
Sim, ao trocar a imagem, o logo será alterado, porém você deverá manter as mesmas dimensões.

Se quiser colocar uma imagem com dimensões diferentes você terá que alterar o CSS.

**Comment by thadeu220** [comment]
ok... muito obrigado, mas, quais são as dimensões do logo atual ?
e uma outra coisa, há como trocar, no nome do site, ou seja, na barra de título do navegador, o nome de mapOS para algo do tipo
"loja blind cell"
substituir o nome mapOS por um nome personalizado no título ?
sei que isso é em um <title> no html, ou algo do tipo.
como proceder ?
grato!

**Comment by thadeu220** [comment]
bem, mas, onde que eu encontro essas metatags title ?
fica em qual caminho?
fica na pasta config? na pasta applications, na assets?
qual o caminho para que eu encontre os arquivos onde tem um <title>
por exemplo   /applications/pages/mapos.php
e lá teria um <title>oi sou um título</title>
mas, onde que estão os arquivos que contem essas metatags?
editar eu sei, mas, eu não tenho paciência.
PHP e code igniter deveria ser um 1,2,3 da programação, mas, tá mais para uma equação a quarto grau.
ajudem o cego!

**Comment by Pr3d4dor** [comment]
No arquivo application/views/tema/topo.php

Se quiser alterar o rodapé fica em application/views/tema/rodape.php

**Comment by thadeu220** [comment]
valeeeeuu!!!!! te amo!!!!!!
kkkkkk!
sério, numa boa, quero pegar você e apertar kkkkk
to brincando, e não sou isso que você tá pensando... kkkk
tom muito feliz já!

**Comment by Pr3d4dor** [comment]
Desculpa, o rodapé fica no arquivo do topo.php ainda

Linhas 209 à 213.

```
<!--Footer-part-->
<div class="row-fluid">
  <div id="footer" class="span12"> <a href="https://github.com/RamonSilva20/mapos" target="_blank"><?php echo date('Y'); ?> &copy; MAP OS - Ramon Silva </a></div>
</div>
<!--end-Footer-part-->
```

Arquivo application/views/tema/topo.php

**Comment by thadeu220** [comment]
valeu meu amor, como que eu faço pra casar com você????? kkkkkkkk
valeu mesmio...
e não, não sou gay!
kkkk

**Comment by JeizonSilva** [comment]
Boa tarde, sabe me informar qual a linha para alterar as dimensões da logo?

**Comment by AirtonFranco** [comment]
linha 13 da login.php

**Comment by xavierfilho** [comment]
Boa Tarde, Tudo bem?
Alguém sabe me dizer como eu altero a cor da entrada do login e senha?

**Comment by Kraniows** [comment]
Olá, amigo não estou conseguindo mudar a logo

**Comment by Pr3d4dor** [comment]
@Kraniows qual erro está ocorrendo?

**Comment by goldenmanutencao** [comment]
> Caros desenvolvedores. alterei o arquivo da logo mas a logo MapOS continua, vocês podem me ajudar com isso? onde altero as dimensões?

Têm que remover os dados do navegador para conseguir ver a nova logo pois fica armazenado a imagem no próprio navegador.
